hack/build

```diff
@@ -8,7 +8,7 @@ GOPATH=$BUILDPATH
 
 
 # Go parameters
-VERSION="0.5.0"
+VERSION="1.0.0-rc1"
```

Comment:
how about 1.0.0-rc1-dev ?  after deadline to release, then we change the version to 1.0.0-rc1

Reply:
since we will use rc2, we don't need dev in the version 😄